Tests/Pcap++Test/Tests/XdpTests.cpp

```diff
@@ -0,0 +1,304 @@
+#include "../TestDefinition.h"
+#include "../Common/GlobalTestArgs.h"
+#include "PcapLiveDeviceList.h"
+#include "XdpDevice.h"
+#include "PcapFileDevice.h"
+#include "Packet.h"
+#include "Logger.h"
+
+
+extern PcapTestArgs PcapTestGlobalArgs;
+
+#if USE_XDP
```

Comment:
better to use ifdef as we set USE_XDP with add_definitions(-DUSE_XDP) 
And we don't explictly set it to 1

Reply:
Good catch! Fixed in https://github.com/seladb/PcapPlusPlus/pull/1228/commits/8ee9c83b433f644b89b5bbafc37a4c1620c28d3c